examples/streaming/tauri.conf.json

```
@@ -47,7 +47,7 @@
       }
     ],
     "security": {
-      "csp": "default-src 'self'; media-src stream: https://stream.localhost asset: https://asset.localhost",
+      "csp": "default-src 'self' ipc:; media-src stream: https://stream.localhost asset: https://asset.localhost",
```

does this not also need the windows version?
```suggestion
      "csp": "default-src 'self' ipc: https://ipc.localhost; media-src stream: https://stream.localhost asset: https://asset.localhost",
```